integration/src/main/java/scripts/common/ApiHelpers.java

```diff
@@ -0,0 +1,43 @@
+package scripts.common;
+
+import bio.terra.datarepo.api.JobsApi;
+import bio.terra.datarepo.api.SnapshotsApi;
+import bio.terra.datarepo.client.ApiClient;
+import bio.terra.datarepo.client.ApiException;
+import bio.terra.datarepo.model.JobModel;
+import bio.terra.datarepo.model.SnapshotRequestModel;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+public class ApiHelpers {
+  public static Map<Object, Object> waitForJobSuccess(ApiClient datarepoClient, String jobId)
+      throws ApiException, InterruptedException {
+    var jobApi = new JobsApi(datarepoClient);
+    JobModel job;
+    do {
+      job = jobApi.retrieveJob(jobId);
+      TimeUnit.SECONDS.sleep(5);
+    } while (job.getJobStatus() == JobModel.JobStatusEnum.RUNNING);
+
+    if (job.getJobStatus() != JobModel.JobStatusEnum.SUCCEEDED) {
+      throw new RuntimeException("Job failed: " + job);
+    }
+    return (Map<Object, Object>) jobApi.retrieveJobResult(jobId);
```

Comment:
I believe this will generate a warning about an unchecked cast. We want the code to compile without any warning, so a `SupressWarnings` is needed.
```suggestion
    @SuppressWarnings("unchecked")
    return (Map<Object, Object>) jobApi.retrieveJobResult(jobId);
```